libraries/ESP8266WebServer/src/ESP8266WebServer.h

```diff
@@ -192,11 +196,28 @@ class ESP8266WebServerTemplate
 
   static String responseCodeToString(const int code);
 
+  void addHook (Hook_f hook)
+  {
```

Comment:
Might be a bit too anal, but can we keep the cuddle consistent in the file?  Everything else has opening bracket on the fcn. line.